testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java

```diff
@@ -32,6 +31,7 @@
 
   private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
 
+  // Align this with testkit/src/main/resources/reference.conf
```

Comment:
Super off-topic: it'd be great if we had a comment indicating `CP` stands for `Confluent Platform`

Reply:
I'm talking about the name of this constant: `DEFAULT_CP_PLATFORM_VERSION`

Reply:
That's an artifact of this type which was from https://github.com/testcontainers/testcontainers-java and modified for our purposes.